python/ray/dag/dag_node.py

```diff
@@ -389,6 +409,27 @@ def traverse_and_apply(self, fn: "Callable[[DAGNode], T]"):
 
         while queue:
             node = queue.pop(0)
+            if node._args_contains_nested_dag_node:
```

Comment:
wrap this block in a method?

Reply:
updated https://github.com/ray-project/ray/pull/48302/commits/adace80b33d0d789185204ab103f899cca0affea